docs/apps/graph-ql-caching.md

```diff
@@ -0,0 +1,19 @@
+---
+sidebar_position: 3
+title: GraphQl Caching
+---
+
+## Macrometa GDN
+
+| **Platform**                       | **Tenat**                      |
+| ---------------------------------- | ------------------------------ |
+| [URL](https://cache.eng.macrometa.io/) | `demo@macrometa.com` |
+
+## Deployment
+
+| **Source Code**                       | **Deployment**                      | 
+| ---------------------------------- | ------------------------------ | 
+| [Github](https://github.com/Macrometacorp/parsec-graphql-cache) | [Deployment](https://macrometacorp.github.io/parsec-graphql-cache/) | 
+
+## Solution
+Immerse yourself in a comprehensive demonstration that illuminates the tremendous strides made in optimizing GraphQL query response times while effectively alleviating the burden on the GraphQL origin server. Our demo showcases a significant architectural enhancement achieved by customizing the Quell Library (https://github.com/open-source-labs/Quell) to leverage GDN KV collection as a superior caching mechanism, surpassing traditional Redis server implementations. By meticulously configuring a robust 4-region US federation with GDN and deploying a single Elasticsearch instance strategically located in the US West region, we unlock unparalleled performance gains. Prepare to delve deep into the intricacies of this cutting-edge setup, unraveling the inherent power of GDN in revolutionizing GraphQL workflows.
```

Comment:
@dlozina-macrometa can we please remove setup details from solution text?
```
Remove:
By meticulously configuring a robust 4-region US federation with GDN and deploying a single Elasticsearch instance strategically located in the US West region,
```

Reply:
@tushar-macrometa sure!